docs/how_to_guides/creating_and_editing_expectations/how_to_create_expectations_that_span_multiple_batches_using_evaluation_parameters.rst

```diff
@@ -0,0 +1,139 @@
+            expectation_suite_name='my_expectation_suite_1'
+        )
+
+        downstream_batch = context.get_batch(
+            context.build_batch_kwargs("my_datasource", "my_generator_name", "my_data_asset_name_2"),
+            expectation_suite_name='my_expectation_suite_2'
+        )
+
+#. **Define an Expectation using an Evaluation Parameter on the downstream Batch.**
+
+    .. code-block:: python
+
+        eval_param_urn = 'urn:great_expectations:validations:my_expectation_suite_1:expect_table_row_count_to_be_between.result.observed_value'
+        downstream_batch.expect_table_row_count_to_equal(
+            value={
+                '$PARAMETER': eval_param_urn, # this is the actual parameter we're going to use in the validation
+                '$PARAMETER.' + eval_param_urn: 10  # this is a *temporary* value so we can execute the notebook
+            }
+        )
+    
+    The core of this is a ``$PARAMETER : URN`` pair. When Great Expectations encounters a ``$PARAMETER`` flag during validation, it will replace the ``URN`` with a value retrieved from an :ref:`Evaluation Parameter Store` or :ref:`Metrics Store`.
+    
+    This declaration above includes two ``$PARAMETERS``. The first is the real parameter that will be used after the Expectation Suite is stored and deployed in a Validation Operator. The second parameter supports immediate evaluation in the notebook.
+
+    When executed in the notebook, this Expectation will generate an :ref:`Expectation Validation Result`:
+
+    .. code-block:: python
+
+        {
+            "result": {
+                "observed_value": 506
+            },
+            "meta": {},
+            "exception_info": null,
+            "success": false
+        }
+
+    .. warning::
+
+        Your URN must be exactly correct in order to work in production. Unfortunately, successful execution at this stage does not guarantee that the URN is specified correctly and that the intended parameters will be available when executed later.
+        
+#. **Save your Expectation Suite**
+
+    .. code-block:: python
+
+        downstream_batch.save_expectation_suite(discard_failed_expectations=False)
+
+    This step is necessary, because your ``$PARAMETER`` will only function properly when invoked within a Validation operation with multiple Batches, the simplest way to execute such an operation invoke is through a :ref:`Validation Operator`, and Validation Operators are configured to load Expectation Suites from Expectation Stores, not memory.
+
+#. **Execute an existing Validation Operator on your upstream and downstream batches.**
+
+    You can do this within your notebook by running ``context.run_validation_operator``:
+
+    .. code-block:: python
+
+        results = context.run_validation_operator(
+            "action_list_operator",
+            assets_to_validate=[
+                upstream_batch,
+                downstream_batch,
```

Comment:
The upstream_batch, downstream_batch parameters should be replaced with batch_kwargs, expectation_suite_name tuples.

Reply:
This executes successfully as written. Are you saying that we want to deprecate this usage of `run_validation_operator` in favor of a required list of batch_kwarg-suite tuples?

Reply:
Yes; in general a validation operator run should be expected to happen in a slim way, without requiring the user to have separately instantiated the batches.

Reply:
Got it. I can make that change.

Observation: this reminds me of a conversation from last year that I don't think we ever finished: boundary methods in the Data Context. Which classes/methods require slim, fully typed and configured objects, and which ones attempt to parse complicated messy things?

Reply:
I was able to successfully implement this change, but I haven’t pushed it because I’m concerned that it makes the instructions for this guide more confusing. ("Why are we re-instantiating all the same objects we instantiated at the beginning of the guide?")

There’s also the lingering problem of “Why do I need to save my Expectation Suite before running my Validation Operator? I already have it in memory.”

Related: I really don’t like that our currently dev loop that forces you to save an Expectation Suite before you’ve tested it. In my mind, that’s a clear anti-pattern. IMO, that alone is enough to insist that ValidationOperators provide a mechanism for running using already-instantiated objects.

Based on those considerations, I think that `run_validation_operator` should support execution using already-instantiated objects, as well as instantiating objects based on new parameters. I think that's different from your view---we should get together to synthesize. 

IMO, the problem with the current implementation is that it (apparently) mixes the two in a confusing way: when you pass an instantiated Batch (with an instantiated Suite), it somehow re-instantiates the Suite itself from disk before executing.

I’m confused by this behavior, tbh. I spent some time digging into the `ValidationOperator` code, and it’s not obvious to me why it’s behaving this way. My instinct is that it’d be worth writing some unit tests for `ActionListValidationOperator._build_batch_from_item` to sanity check its behavior when passed different object types.

Reply:
Okay, I pushed my changes so that I can switch branches. But I'm not sure that this is really how we want it to work.

Reply:
Somehow I missed that you'd responded on this. I'm going to respond but merge since the changes are already made...

Quick inlines since I think there is more misunderstanding than problem:

1. Why are we re-instantiating all the same objects we instantiated at the beginning of the guide? -> Because we're showing the code that would be run in a pipeline context (instead of the notebook only). 

2. Why do I need to save my Expectation Suite before running my Validation Operator? -> You don't. As you pointed out, the previous iteration of code *would work*, but it's not the recommended path precisely because the validation operator is *primarily intended* to be used for *validation* not for iterative development.

3. I really don’t like that our currently dev loop that forces you to save an Expectation Suite before you’ve tested it. In my mind, that’s a clear anti-pattern. IMO, that alone is enough to insist that ValidationOperators provide a mechanism for running using already-instantiated object. -> It doesn't. Again (a) you can run the validation operator from a batch without a saved expectation suite (but that's not what this guide is showing, and (b) you can test an expectation suite without a validation operator (ditto).

4. I think that run_validation_operator should support execution using already-instantiated objects, as well as instantiating objects based on new parameters. -> I agree...but as above I think it's not the "recommended" path.

5. IMO, the problem with the current implementation is that it (apparently) mixes the two in a confusing way: when you pass an instantiated Batch (with an instantiated Suite), it somehow re-instantiates the Suite itself from disk before executing. -> I think this is inaccurate. If you pass an instantiated Batch, it currently uses the attached Suite.